test/plugins/link.js

```diff
@@ -26,7 +29,7 @@ describe('Link plugin', function() {
     });
 
     this.irc.createMessage({
-      message: 'http://localhost:9002/basic'
+      message: 'http://127.0.0.1:9002/basic'
```

Comment:
https://github.com/erming/shout/blob/master/src/plugins/irc-events/link.js#L21 makes it look it ignores `https?://localhost` files